reference/3.0/Microsoft.PowerShell.Core/About/About.md

```diff
@@ -17,445 +17,337 @@ About topics cover a ranges of concepts about PowerShell.
+Describes the features of the Alias provider.
 
 ### [about_Aliases](about_Aliases.md)
-
 Describes how to use alternate names for cmdlets and commands in PowerShell.
 
 ### [about_Arithmetic_Operators](about_Arithmetic_Operators.md)
-
 Describes the operators that perform arithmetic in PowerShell.
 
 ### [about_Arrays](about_Arrays.md)
-
 Describes arrays, which are data structures designed to store collections of items.
 
 ### [about_Assignment_Operators](about_Assignment_Operators.md)
-
 Describes how to use operators to assign values to variables.
 
 ### [about_Automatic_Variables](about_Automatic_Variables.md)
-
 Describes variables that store state information for PowerShell. These variables are created and maintained by PowerShell.
 
 ### [about_Break](about_Break.md)
-
 Describes a statement you can use to immediately exit Foreach, For, While, Do, or Switch statements.
 
 ### [about_CimSession](about_CimSession.md)
-
 Describes a CimSession object and the difference between CIM sessions and PowerShell sessions.
 
 ### [about_Command_Precedence](about_Command_Precedence.md)
-
 Describes how PowerShell determines which command to run.
 
 ### [about_Command_Syntax](about_Command_Syntax.md)
-
 Describes the syntax diagrams that are used in PowerShell.
 
 ### [about_Comment_Based_Help](about_Comment_Based_Help.md)
-
 Describes how to write comment-based help topics for functions and scripts.
 
 ### [about_CommonParameters](about_CommonParameters.md)
-
 Describes the parameters that can be used with any cmdlet.
 
 ### [about_Comparison_Operators](about_Comparison_Operators.md)
-
 Describes the operators that compare values in PowerShell.
 
 ### [about_Continue](about_Continue.md)
-
 Describes how the `Continue` statement immediately returns the program flow to the top of a program loop.
 
 ### [about_Core_Commands](about_Core_Commands.md)
-
 Lists the cmdlets that are designed for use with PowerShell providers.
 
 ### [about_Data_Sections](about_Data_Sections.md)
-
 Explains Data sections, which isolate text strings and other read-only data from script logic.
 
 ### [about_Debuggers](about_Debuggers.md)
-
-Describes the Windows PowerShell debugger.
+Describes the PowerShell debugger.
 
 ### [about_Do](about_Do.md)
-
 Runs a statement list one or more times, subject to a While or Until condition.
 
 ### [about_Environment_Provider](about_Environment_Provider.md)
-
-Provides access to the Environment variables and the values that they represent.
+Describes the features of the Environment provider.
 
 ### [about_Environment_Variables](about_Environment_Variables.md)
-
-Describes how to access Windows environment variables in Windows PowerShell.
+Describes how to access Windows environment variables in PowerShell.
 
 ### [about_Eventlogs](about_Eventlogs.md)
-
 Windows PowerShell creates a Windows event log that is named "Windows PowerShell" to record Windows PowerShell events.
```

Comment:
Should clean up this 'Windows', too.

Reply:
This is definately about Windows PowerShell. And that is the name of the Event log and the event source.

Reply:
That's fair.